content/SCALE/SCALETutorials/Apps/Syncthing.md

```diff
@@ -0,0 +1,165 @@
+{{< /enterprise >}}
+
+Syncthing is tested and validated to work in harmony with TrueNAS platforms and underlying technologies such as ZFS to offer a turnkey means of keeping data across many systems. It can seamlessly integrate with TrueNAS.
+
+Syncthing is a file synchronization application that provides a simple and secure environment for file sharing between different devices and locations.  
+Use it to synchronize files between different departments, teams, or remote workers.
+
+Syncthing does not use or need a central server or cloud storage. 
+All data is encrypted and synchronized directly between devices to ensure files are protected from unauthorized access.
+
+Syncthing is easy to use and configure. 
+You can install on a wide range of operating systems, including Windows, macOS, linux, FreeBSD, iOS or Android. 
+The Synchthing web UI provides users with easy management and configuration of the application software.
+
+{{< expand "How does Syncthing work?" "v" >}}
+Syncthing does not have a central directory or cache to manage. 
+It segments files into pieces called blocks. 
+These blocks transfer data from one device to another. 
+Multiple devices can share the synchronization load in a similar way to the torrent protocol. 
+With more devices and smaller blocks, devices recieve data faster because all devices fetch blocks in parallel.
+
+Syncthing renames files and updates metadata more efficiently because renaming does not cause a retransmission of that file.
+
+Temporary files store partial data downloaded from devices. 
+Temporary files are removed when a file transfer completes or after the configured amount of time elapses.
+{{< /expand >}}
+
+## Before You Begin
+
+You can add a dataset for Syncthing storage (for example, *syncthing*) before you install the application or create the dataset through the application installation wizard.
+
+Decide on a consistent file-naming convention to avoid conflict situations where data does not or cannot synchronize because of file name conflicts.
+Path and file names in the Syncthing application are case sensitive. 
+Creating a file named MyData.txt is not the same mydata.txt file in Syncthing. 
+
+If not already complete, set the pool for applications to use.
+
+## Installing the Syncthing Application
+
+Go to **Apps > Available Applications**, locate the **Syncthing** application widget. 
+
+{{< trueimage src="/images/SCALE/22.12/SyncthingAppWidget.png" alt="Syncthing Application Widget" id="1: Syncthing Application Widget" >}}
+
+Click **Install** to open the **sycnthing** configuration wizard.
+
+Accept the default value or enter a name in **Application Name**.
+
+{{< trueimage src="/images/SCALE/22.12/SyncthingApplicatonName.png" alt="Syncthing Application Name and Version" id="2: Syncthing Application Name and Version" >}}
+
+Select the timezone where the TrueNAS server is located from the **Timezone** dropdown list.
```

Comment:
do not see this (Timezone) in the wizard or is talking about another window and or next steps.